modules/http/src/main/java/org/mule/module/http/internal/listener/DefaultHttpListenerConfig.java

```diff
@@ -334,4 +327,8 @@ public void setConnectionIdleTimeout(int connectionIdleTimeout)
         this.connectionIdleTimeout = connectionIdleTimeout;
     }
 
+    public void setConnectionManager(HttpListenerConnectionManager connectionManager)
```

Comment:
what implications? Making it mutable for testing means that it can also be mutable in production